opencti-platform/opencti-front/src/utils/graph/EntityDetails.tsx

```diff
@@ -343,11 +347,22 @@ EntityDetailsComponentProps
       <Typography variant="h3" gutterBottom={true} className={classes.label}>
         {t('Description')}
       </Typography>
-      {stixCoreObject.description && stixCoreObject.description.length > 0 ? (
-        <ExpandableMarkdown source={stixCoreObject.description} limit={400} />
+      {entityDescription && entityDescription.length > 0 ? (
+        <ExpandableMarkdown source={entityDescription} limit={400} />
       ) : (
         '-'
       )}
+      {!stixCoreObject.parent_types.includes('Stix-Cyber-Observable')
+        && (<div>
+          <Typography variant="h3" gutterBottom={true} className={classes.label}>
+            {t('Confidence level')}
+          </Typography>
+          {stixCoreObject.confidence
+            ? <ItemConfidence confidence={stixCoreObject.confidence} entityType="stix-core-object"/>
+            : ('-')
```

Comment:
Is it possible to use the "FieldOrEmpty" component here ?

Reply:
fixed